go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/btcsuite/btcwallet/walletdb v1.4.0
 	github.com/btcsuite/btcwallet/wtxmgr v1.3.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/dchest/blake2b v1.0.0
```

Comment:
I'm wondering about golang.org/x/crypto/blake2b.  It's maintained and has architecture specific implementations and assembly (may or may not be viewed a good thing though)

Reply:
I did look try this, and couldn't get tests to pass using our personalization keys. Not sure if key handling is different somehow.

Reply:
Oh... https://github.com/golang/go/issues/32447
personalization != MAC key